scripts/pi-hole/js/queries.js

```diff
@@ -21,7 +21,8 @@ var replyTypes = [
   "SERVFAIL",
   "REFUSED",
   "NOTIMP",
-  "upstream error"
+  "upstream error",
```

Comment:
Total aside, I read a codestyle rule that said you should always put a comma after the last element in multiline array to make it easier to review.

I didn't get it until I just saw this diff. (if the comma was already there then the diff would only show the new stuff on line 25)

Anyway...

Reply:
I actually put the command there but `xo` complained and removed it in `--fix` mode. So we need a style change here to make this work.

Reply:
It actually seems to be a `prettier` rule:

![Screenshot from 2021-06-23 11-18-58](https://user-images.githubusercontent.com/16748619/123071418-ee74d280-d414-11eb-9c97-f0be1be7d503.png)

Reply:
Will be changed by https://github.com/pi-hole/AdminLTE/pull/1838 - please first merge this PR and I'll update here afterwards. Converting to `draft` for now.